Now I don't get any visible check on MacOS and It crashed when I clicked on the 'Write Tags' checkbox too much
```plain
2024-07-27 14:14:05.686 Python[23532:673543] WARNING: Secure coding is not enabled for restorable state! Enable secure coding by implementing NSApplicationDelegate.applicationSupportsSecureRestorableState: and returning YES.
2024-07-27T14:14:23 | comictagger | CRITICAL | Uncaught exception: AttributeError: 'NoneType' object has no attribute 'checkState'
Traceback (most recent call last):
  File "/Users/timmy/build/source/comictagger/comictaggerlib/ui/customwidgets.py", line 144, in eventFilter
    self.toggleItem(index.row())
  File "/Users/timmy/build/source/comictagger/comictaggerlib/ui/customwidgets.py", line 205, in toggleItem
    if self.model().item(index).checkState() == Qt.Checked:
AttributeError: 'NoneType' object has no attribute 'checkState'
```

--- a/comictaggerlib/ui/customwidgets.py
+++ b/comictaggerlib/ui/customwidgets.py
@@ -4,7 +4,6 @@ from __future__ import annotations
 
 from enum import auto
 from sys import platform
-import platform
 from typing import Any
 
 from PyQt5 import QtGui, QtWidgets

